docs/advanced/bootloader.md

```diff
@@ -94,3 +98,18 @@ It produces the following ROM layout:
 ``` 
 
 For an example showing how to create an application that uses a bootloader, see the [mbed OS bootloader example](https://github.com/armmbed/mbed-os-example-bootloader-blinky) repository.
+
+## Unmanaged bootloader
+
+You want to have an unmanaged bootloader when your bootloaders requirements conflict with the requirements of the managed bootloader. You need an unmanaged bootloader when your bootloader does not come before your application in ROM or your application does not start immediately after your bootloader. Unlinke a managed bootloader, an unmananged bootloader does not automatically merge the bootloader image with the application image after building the application. We expect a user of an unmanaged bootloader build to construct there own set of scripts built atop the `mbed compile` primitive to perform bootloader and application merging.
```

Comment:
- In first sentence, please change "bootloaders" to "bootloader's" to show possession.
- In third sentence, please change "Unlinke" to "Unlike" to fix the typo.
- Please change the start of the last sentence to "We expect users of unmanaged bootloader builds to construct their own sets of scripts built atop..." for agreement.